### PR TITLE
STY: Enable ruff ambiguous unicode character

### PR DESCRIPTION
**Pull request by mroeschke**

### CODE REVIEW

File: pandas/tests/frame/methods/test_to_csv.py [hunk 1]
```diff
@@ -957,7 +957,7 @@ def test_to_csv_path_is_none(self, float_frame):
             (DataFrame([["abc", "def", "ghi"]], columns=["X", "Y", "Z"]), "ascii"),
             (DataFrame(5 * [[123, "你好", "世界"]], columns=["X", "Y", "Z"]), "gb2312"),
             (
-                DataFrame(5 * [[123, "Γειά σου", "Κόσμε"]], columns=["X", "Y", "Z"]),
```
Comment (rhshadrach) [inline pandas/tests/frame/methods/test_to_csv.py:960]: noqa here instead?

File: pandas/tests/series/methods/test_to_csv.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def test_to_csv_path_is_none(self):
             # GH 21241, 21118
             (Series(["abc", "def", "ghi"], name="X"), "ascii"),
             (Series(["123", "你好", "世界"], name="中文"), "gb2312"),
-            (Series(["123", "Γειά σου", "Κόσμε"], name="Ελληνικά"), "cp737"),
```
Comment (rhshadrach) [inline pandas/tests/series/methods/test_to_csv.py:125]: Ditto

File: pandas/tests/strings/test_strings.py [hunk 3]
```diff
@@ -226,8 +226,8 @@ def test_isnumeric_unicode(method, expected, any_string_dtype):
     # 0x00bc: ¼ VULGAR FRACTION ONE QUARTER
     # 0x2605: ★ not number
     # 0x1378: ፸ ETHIOPIC NUMBER SEVENTY
-    # 0xFF13: ３ Em 3
-    ser = Series(["A", "3", "¼", "★", "፸", "３", "four"], dtype=any_string_dtype)
+    # 0xFF13: ３ Em 3  # noqa: RUF003
+    ser = Series(["A", "3", "¼", "★", "፸", "3", "four"], dtype=any_string_dtype)
```
Comment (rhshadrach) [inline pandas/tests/strings/test_strings.py:230]: Seems like the comment above is indicating this one shouldn't be changed?

Comment (rhshadrach) [reply pandas/tests/strings/test_strings.py:230]: Same remark for the rest of the changes in this file.

**Review by rhshadrach — CHANGES_REQUESTED** [review]
Nice - never even thought about these sneaking in here when reviewing a PR.

File: pandas/tests/frame/methods/test_to_csv.py (hunk 1 above)
Comment (mroeschke) [reply pandas/tests/frame/methods/test_to_csv.py:960]: Good call. Fixed

File: pandas/tests/series/methods/test_to_csv.py (hunk 2 above)
Comment (mroeschke) [reply pandas/tests/series/methods/test_to_csv.py:125]: Yeah good call. Fixed

File: pandas/tests/strings/test_strings.py (hunk 3 above)
Comment (mroeschke) [reply pandas/tests/strings/test_strings.py]: Good catch. Added noqas here

**Review by rhshadrach — APPROVED** [review]
lgtm

**Comment by rhshadrach** [comment]
Thanks @mroeschke

**Pull request merged**